src/ppc.rs

```diff
@@ -1,13 +1,21 @@
+//! Support for floating point types that use PowerPC semantics.
+
 use crate::ieee;
 use crate::{Category, ExpInt, Float, FloatConvert, ParseError, Round, Status, StatusAnd};
 
 use core::cmp::Ordering;
 use core::fmt;
 use core::ops::Neg;
 
+/// A larger floating point number represented by two smaller floats.
 #[must_use]
 #[derive(Copy, Clone, PartialEq, PartialOrd, Debug)]
 pub struct DoubleFloat<F>(F, F);
+
+/// 128-bit floating point number comprised of two IEEE [`Double`](ieee::Double) values.
+///
+/// This is the "IBM Extended Double" format, described at
+/// <https://www.ibm.com/docs/bg/aix/7.1?topic=sepl-128-bit-long-double-floating-point-data-type>.
```

Comment:
```suggestion
/// <https://www.ibm.com/docs/en/aix/7.3?topic=sepl-128-bit-long-double-floating-point-data-type>.
```

Version bump and Bulgarian to English (didn't affect the actual content, only some parts of the website).